nss-day-cohort-48/giffygram-jumpstation
# Smb recieving messages

#### Changes Made

1. Modified file `DirectMessages` to include `Ability to see received messages` and functionality.
   ​

#### Steps to Review

1. Checkout this branch locally.
   ```
   git fetch --all
   git checkout smb-recieving-messages
   ```
2. Open a new Terminal tab (⌘T) and navigate to the server directory.
3. Test app functionality.
   > Verify the notifications icon shows the correct number of messages in your database.
   >Open the messages and verify they show up properly on the screen. 
4. View code file.
   > Confirm file modifications are present as indicated above.
   > Confirm no unused code or extraneous comments exist.
